### PR TITLE
add loading="lazy" to detail page screenshot thumbs

### LINKED ISSUE #10094: Investigate detail page previews optimizations
We spend a significant time loading preview images when loading a detail page (https://www.webpagetest.org/result/210212_DiDG_22aa7b4334f8b6993f9a754cff680fbb/1/details/). Some ideas to explore:
- Lazy loading ? Notably, previews from detail page are all loaded early even though some are behind scroll
- Smaller thumbnails ? We load a 640x480 thumbnail that is displayed at 320x240. It looks good on fancy retina displays, but is this necessary ?
- More aggressive image optimization techniques or different format (WebP?)

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #10094

### CODE REVIEW

File: src/amo/components/ScreenShots/index.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ export default class ScreenShots extends React.Component<Props> {
                       src={preview.thumbnail_src}
                       width={preview.thumbnail_w}
                       height={preview.thumbnail_h}
+                      loading="lazy"
```
Comment (willdurand) [inline src/amo/components/ScreenShots/index.js:82]: Adding an assertion for this would be good.

Comment (eviljeff) [reply src/amo/components/ScreenShots/index.js:82]: like, an `expect` line in the tests?  Or something else?

Comment (willdurand) [reply src/amo/components/ScreenShots/index.js:82]: yes an `expect` to prevent the future AMO team to remove this prop because it's useless :p

**Review by diox — APPROVED** [review]

File: src/amo/components/ScreenShots/index.js (hunk 1 above)
Comment (eviljeff) [reply src/amo/components/ScreenShots/index.js:82]: https://github.com/mozilla/addons-frontend/pull/10189/commits/eafe9ecfd3574b68b647187553d8d05bc9082ff0

**Pull request merged**